compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -149,17 +149,19 @@ object Splicer {
       try clazz.getMethod(name.toString, paramClasses: _*)
       catch {
         case _: NoSuchMethodException =>
-          val msg = s"Could not find interpreted method ${clazz.getCanonicalName}.$name with parameters $paramClasses"
+          val msg = s"Could not find inline macro method ${clazz.getCanonicalName}.$name with parameters $paramClasses $extraMsg"
           throw new StopInterpretation(msg, pos)
       }
     }
 
+    private def extraMsg = "(the most common reason for that is that you cannot use inline macro implementations in the same compilation run that defines them)" 
```

Comment:
Maybe make this sentence shorter as it is in parenthesis. Or make it a separate sentence.

Something like "You defined and used macros in the same compilation run?"

Reply:
Splitted into two sentences

Reply:
I will keep this text as it is the same as for scala 2 macros andif they seach for it they might still get some useful insight on why they need to be in different compilation units.